cmd/sriov-network-config-daemon/main.go

```diff
@@ -44,5 +45,6 @@ func init() {
 func main() {
 	if err := rootCmd.Execute(); err != nil {
 		log.Log.Error(err, "error executing sriov-network-config-daemon")
+		os.Exit(1)
```

Comment:
This change submitted as a separate PR https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/993